roles/openshift_facts/tasks/main.yml

```diff
@@ -22,6 +22,7 @@
   openshift_facts:
     role: common
     local_facts:
+      debug_level: "{{ openshift_debug_level | default(2) }}"
```

Comment:
Tested this out locally on a cluster I already had up and it changed the value from 2 to 5. I consider that a victory for the commonwealth.